.github/workflows/upload-to-pypi.yml

```
@@ -45,11 +45,19 @@ jobs:
           cp setup.cfg testdir/
           cd testdir/
           pytest --showlocals --durations=10 --pyargs --cov-report=xml --cov=sktime sktime
-      - name: Upload to PyPI
+      - name: Upload wheel to PyPI
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
           ls -l dist/*
           python -m pip install --upgrade twine
           python -m twine upload -r pypi --non-interactive --skip-existing --verbose dist/*.whl
```

Would this work too? 
```suggestion
          python -m twine upload -r pypi --non-interactive --skip-existing --verbose dist/*
```

---

@mloning yes this is a fine simplification! I figured it would be nice to speed up the job by avoiding the duplicate uploads, but simpler code is also great :) I'll add this suggestion now.